README.adoc

```diff
@@ -1,13 +1,25 @@
 # Grails Testing Support
 
-image:https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml/badge.svg["Java CI", link: "https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml"]
+image:https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A["Revved up by Develocity", link: "https://ge.grails.org/scans"]
+image:https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml/badge.svg?event=push["Java CI", link: "https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml"]
+image:https://github.com/grails/grails-testing-support/actions/workflows/groovy-joint-workflow.yml/badge.svg?event=push["Groovy Joint Validation Build", link: "https://github.com/grails/grails-testing-support/actions/workflows/groovy-joint-workflow.yml"]
```

Comment:
The links on the badges do not seem to work as expected.

Reply:
Sorry, I should have checked this locally instead of just copying the existing one 3 times and updating on github.  All working now.